src/math/Matrix4.js

```diff
@@ -117,10 +117,12 @@ THREE.Matrix4.prototype = {
 
 	extractRotation: function () {
 
-		var v1 = new THREE.Vector3();
+		var v1;
 
 		return function ( m ) {
 
+			if ( v1 === undefined) v1 = new THREE.Vector3();
```

Comment:
if ( v1 === undefined ) v1 = new THREE.Vector3();

If you like, this would be a nice test for http://zz85.github.io/mrdoobapproves/ to find the remaining formatting issues. BTW, it also fails to correct blank lines with leading spaces, or blank lines with tabs only.